src/server-workers.js

```diff
@@ -0,0 +1,16 @@
+import * as system from "./system.js";
+import "./helpers/workers/config.js";
+import { handleRequest } from "./index.js";
+
+((main) => {
+  system.sub("go", systemUp);
+})();
+
+function systemUp() {
+  if (typeof addEventListener === "undefined") {
+    throw new Error("workers env missing addEventListener");
```

Comment:
this check is no longer required, this file is only for workers.

Reply:
Explicit is better than implicit, especially for entry points which are outside the scope of the bundler/linter/transpilers.

That is, this preserves the API between runners (docker run) and this entrypoint

Reply:
It will throw error regardless!?

Reply:
You mean, this code throws `err` in Workers too? I don't follow...

Reply:
If it is `undefined` it will be Ref. Err. anyway, why check it to throw again?

Reply:
explicit is better than implicit...?